bindings_generator/src/api.rs

```diff
@@ -93,6 +98,15 @@ pub struct Enum {
     pub values: HashMap<String, i64>,
 }
 
+#[derive(Deserialize, Debug)]
+pub struct Property {
+    pub name: String,
+    pub r#type: String,
```

Comment:
I believe the accepted practice is to add a trailing underscore to keyword identifiers? i.e. `type_`. You can then rename the field with `#[serde(rename = "type")]`.